docs/components/cache.rst

```diff
@@ -1,4 +1,122 @@
-Cache
+CACHE
 #####
 
+Symfony makes heavy use of a filesystem cache. Frequently clearing this cache will be required when developing for Mautic. By default, the cache is located in ``app/cache/ENV`` where ``ENV`` is the environment currently accessed (``dev`` or ``prod``). To rebuild the cache, the ``ENV`` can just be deleted or run the Symfony command ``php app/console cache:clear --env=ENV``. If a specific environment is not passed to the command via ``--env=ENV``, the ``dev`` environment will be used by default.
```

Comment:
Update for Mautic 3 and beyond (app/console -> bin/console, app/cache -> var/cache) and multiple Vale fixes.

```suggestion
Symfony makes heavy use of a filesystem cache. When developing for Mautic, clearing the cache is a regular occurrence. By default, Mautic instances have the cache located in ``var/cache/ENV`` where ``ENV`` is the environment currently accessed (``dev`` or ``prod``). To rebuild the cache, delete the relevant ``ENV`` folder within the cache directory, or run the Symfony command ``php bin/console cache:clear --env=ENV``. If a specific environment isn't passed to the command via ``--env=ENV``, Mautic uses the ``dev`` environment by default.
```